### ISSUE TITLE
The addon not being recognized in Windows

### ISSUE THREAD

**Issue by CreativeBlends** [Secrop/ShaderNodesExtra2.80#11]
I have been trying to install the addon and the blender says that the module is installed but it does not come in the addon sections.

**Comment by Secrop** [comment]
Did you download the 'Release' file (https://github.com/Secrop/ShaderNodesExtra2.80/releases/tag/0.2.0) or the "Source as Zip"?
The later will fail if you try to load the zip file directly into blender (the folder name inside the zip is incompatible with the python naming conventions).

**Comment by CreativeBlends** [comment]
I had downloaded the 'Source as Zip'. Should I Download the source code in the First link sent by you. and does this support eevee nodes like 'Shader to RGB' ?

**Comment by Secrop** [comment]
Yes, you should use the Release tag. But if you've downloaded and installed the 'Source as Zip' you can rename the folder name in the [User path](https://docs.blender.org/manual/en/latest/advanced/blender_directory_layout.html#temp-dir) from 'ShaderNodesExtra2.80-master' to simply 'ShaderNodesExtra'.
Yes, it's possible to have Eevee nodes (and with some changes, even Compositor and Geometry nodes)... but this is not fully supported/tested.

**Comment by CreativeBlends** [comment]
I have installed it but it has some problems. it shows the error icon and nothing seems to work. can you send the correct link for the download?

**Comment by Secrop** [comment]
If there's an error icon, then there should be a error printed in the console. Please start Blender from a console, and post the error description here, and also clarify which OS, and Blender version. (Also bare in mind that the addon shows a warning icon, which is not an error, and it's just to clarify that the addon is an alpha version).
The correct link is the one I posted above, thought you can still use the 'Source as Zip' as long you rename the final folder in your user paths (also explained in my previous post)..
In the end, you should be able to see the nodes that are bundled with the addon, when you're working with a Cycles material.
Note that Eevee and Geometry nodes were created after the release of this addon, so there're some changes in code that need to be addressed, such as some poll declaration in the nodes code, that the addon will need to add the node into the correct context.

**Comment by CreativeBlends** [comment]
I have installed it but i don't understand how to use those nodes. there seems to be three nodes : Loop node, displacement bake, and normal bake.

**Comment by Secrop** [comment]
Actually, there are 6 nodes (the other 3 that you don't mention are in the «Converter» sub menu,) and they are lightly described in the end ot the [ReadMe.md](https://github.com/Secrop/ShaderNodesExtra2.80/blob/master/README.md). But as it states in the document, their purpose is to examplify how nodes can be created, so that you can start writing your own nodes.

**Comment by CreativeBlends** [comment]
I wanted to ask if you could make a demo video of how to use them as it would be a great help ?

**Comment by Secrop** [comment]
Sorry, but I don't have time for making videos. Apart from the 'Compare', 'Interpolate' and 'Switch', which are very straigth forward to use, the rest of the nodes require that you understand what you want to do. There's are threads in BA about the other nodes, like the [Loop node](https://blenderartists.org/t/secrops-cycles-loop-node/689357) or the [Displacement Bake](https://blenderartists.org/t/baking-the-perfect-displacement-map-blender-cycles/687770). The Normal Bake, will just convert the normals into a Tangent map (Note: this was created prior to the Cycles ability to bake Normals, so you probably won't be using this unless you want a different TangentSpace).

**Comment by CreativeBlends** [comment]
I have tried using it but there is some error when I click ok after writing the bl name and bl label 
here is the images of the error: 1 - https://drive.google.com/file/d/11S28wyn2fxSxiGrJ30Ag5ZAW95kZGxYh/view?usp=sharing
2 - https://drive.google.com/file/d/1N8i_sh--mwZJIjeXu4fwiPqgwHU-U8Nl/view?usp=sharing

**Comment by Secrop** [comment]
That's not surprising... I haven't changed the Converter's code since I first wrote it 5 years ago.. and the operator won't recognize most of new nodes since then. I could update the operator, but my plan was to update the whole addon first, to adopt all node types currently in blender, thought never had the time to do it. :(

**Comment by CreativeBlends** [comment]
So is there a way right now that can correct it ? if there isn't please let me know the estimated time it will take to complete the update.

**Comment by Secrop** [comment]
If you're ok with python, you can edit and correct the file that was created in the Nodes folder. Hopefully, you just need to check all nodes that are created there, and all links.

If I have time this weekend I'll try to update the addon, or if you send me the blend file with the nodegroup I could take a look at it.

**Comment by CreativeBlends** [comment]
I am not good with python and can't do much. So when you update please tell me.

**Comment by CreativeBlends** [comment]
Secrop can I ask if the addon has been updated cause there are some things I need to do urgently?

**Comment by Secrop** [comment]
@CreativeBlends I did some ground changes to the addon, but haven't had the time to fully tested.. Since it is now working with all kind of nodes (not just shader nodes), I'll probably create a new repository for it. So probably friday night or saturday it will be online.
If that is not suitable for you, send me a blend file with the nodegroup to <porces at gmail dot com>, and I'll try to fix it late today.
